### DISCUSSION TITLE
I have a URL problem when running the Filament project

### DISCUSSION THREAD

**Discussion by yoe174** [filamentphp/filament#13617] [Help]
hello my name is natan. I've just learned to code in the last few months and I'm trying to make a simple project using Laravel and Filament version 3.2. I want to ask about this filament because I found a problem that made me a little confused, namely when running this filament project. I use Laragon to run this program and I have compared two ways or methods when logging in to the Filament project, namely the first way using a URL like in general "http://localhost/myproject/public/admin/login" and the second using PHP Artisan Server "http://127.0.0.1:8000/admin/login". My question is why only the second method can log into my filament project while the first method has a problem after logging in "The POST method is not supported for route admin/login. Supported methods: GET, HEAD.". So as long as I use Filament, I always use PHP Artisan when I want to run my project and I'm worried that I will have a little problem when I deploy it to hosting. Is this special filament that can only be used in PHP Artisan Server and cannot be accessed using a regular URL? can you help me or give advice regarding this problem on how to use a regular url without php artisan serve or is there any sourcode that needs to be changed? sorry if my question is a small change and trivial question because I am still in the learning stage and sorry for my sloppy and bad language because I use google translate

![Screenshot 2024-07-18 001948](https://github.com/user-attachments/assets/f132d86d-19fa-4d0b-9c4e-b48a1f08267d)
![Screenshot 2024-07-18 002058](https://github.com/user-attachments/assets/d146415b-9127-4331-a948-620dd07f65fa)

**Comment by leandrocfe** [comment]
You should map the `public` folder in the laragon as the root path and don't access `/public` in the url, like http://localhost/admin/login or http://yourproject.localhost/admin/login

**Reply by titanaprilian** [reply]
Can you give me the details about this ? I still don't understand what you are talking about
